terraform/modules/iam/inspector_findings.tf

```diff
@@ -0,0 +1,23 @@
+resource "aws_iam_role_policy_attachment" "inspector_lambda_common_policy_attach" {
+  count      = var.inspector == true ? 1 : 0
+  role       = aws_iam_role.inspector_assume_role.*.name[0]
+  policy_arn = aws_iam_policy.lambda_common_policy.*.arn[0]
```

Comment:
Do you think it's worth saving these 3 variables to a local variable block since they are used more than once?

Reply:
It is yeah. You've also made me realise that that's the wrong policy I've put in there. I've moved them to variables.